wordpress-mobile/WordPress-Android
# Add empty screen handling to media picker

This PR adds an empty screen to all the data sources. There are two empty screens for the pexel data source - one is shown before the search and one when the search result is empty. The other two data sources use standard empty screens:

Pexel initial screen:
![Screenshot_1601987969](https://user-images.githubusercontent.com/1079756/95202681-f090f580-07e1-11eb-84f9-3223b663cebe.png)
Empty search results:
![Screenshot_1601987976](https://user-images.githubusercontent.com/1079756/95202689-f38be600-07e1-11eb-9af4-1f10da58babe.png)

To test:
- Try to add media from the stock library
- Notice the empty screen before you enter query
- Enter a query that doesn't return any results
- Notice a different empty screen
- Try search for non-sense queries in the media library and the device library
- Notice the empty screens

PR submission checklist:
